File: 02-Número-Espelhado/solucao.py
def inverter_numero(numero: int) -> int:
    invertido = 0
    while numero > 0:
        digito = numero % 10;
        invertido = (invertido * 10) + digito
        numero = numero // 10
    return invertido

## Casos de teste
if __name__ == "__main__":
    print(f"Invertido 3815: {inverter_numero(3815)}")
    print(f"Invertido 100: {inverter_numero(100)}")
    print(f"Invertido 7: {inverter_numero(7)}")
    

File: 02-Número-Espelhado/solucao.js
function inverterNumero(numero){
    let invertido = 0;
    while(numero > 0){
        let digito = numero % 10;
        invertido = (invertido * 10) + digito;
        numero = Math.floor(numero / 10);
    }

    return invertido;
}

//Casos de teste
console.log(`Invertido 3815: ${inverterNumero(3815)}`);
console.log(`Invertido 100: ${inverterNumero(100)}`);
console.log(`Invertido 7: ${inverterNumero(7)}`);
